index.bs

```diff
@@ -1581,14 +1581,14 @@ DateRemoteValue = {
 
 MapRemoteValue = {
   type: "map",
-  value: MappingRemoteValue,
+  ?value: MappingRemoteValue,
```

Comment:
Should `?value` move after `?internalId` to keep things sorted. I see other definitions seem to follow this pattern (eg ArrayRemoteValue)